### PR TITLE
Def of reduced cohomology without InducedAbGroupPath

### PR DESCRIPTION
**Pull request by aljungstrom**
I am able to define reduced cohomology directly via InducedAbGroup and now do the previous hack with defining it via InducedAbGroupPath. However, is still have to have coHomGroup≡coHomRedGroup'  and coHomGroup≡coHomRedGroup separate. If I remove coHomGroup≡coHomRedGroup' and write the exact same stuff in the definition of coHomGroup≡coHomRedGroup, it takes ages to type check.

### CODE REVIEW

**Comment by Saizan** [comment]
Does `coHomRedGroup` itself get slow to typecheck or later uses?

**Comment by mortberg** [comment]
We figured it out! The solution was to add  `--experimental-lossy-unification`  @Saizan

**Comment by mortberg** [comment]
Without it I think Agda tried to unfold `InducedAbGroup` when trying to fill a goal with `InducedAbGroupPath`. This is a terrible idea and I guess that adding the flag stopped Agda from doing this

**Comment by aljungstrom** [comment]
> Does `coHomRedGroup` itself get slow to typecheck or later uses?

I'm impressed that you managed to understand the description of this PR, because I can barely do so myself...

To answer your question: it actually type checks fast in later uses!

File: Cubical/ZCohomology/Properties.agda [hunk 1]
```diff
@@ -233,23 +233,22 @@ coHomGr≅coHomRedGr : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
...
+  InducedAbGroup (coHomGroup (suc n) (typ A))
+                 (coHomRed (suc n) A , _+ₕ∙_)
+                 (isoToEquiv (invIso (Iso-coHom-coHomRed n)))
+                 (homhelp n A)
 
 abstract
   coHomGroup≡coHomRedGroup : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
                           → coHomRedGroup (suc n) A ≡ coHomGroup (suc n) (typ A)
-  coHomGroup≡coHomRedGroup = coHomGroup≡coHomRedGroup'
+  coHomGroup≡coHomRedGroup n A =
```
Comment (mortberg) [inline Cubical/ZCohomology/Properties.agda:247]: Wait a minute, the name of this lemma is not the same order as in the type. Please swap the type and remove the `sym`

Comment (mortberg) [reply Cubical/ZCohomology/Properties.agda:247]: ping @aljungstrom

Comment (aljungstrom) [reply Cubical/ZCohomology/Properties.agda:247]: wups, forgot about this. It's fixed now.

**Pull request merged**